library/java/src/io/envoyproxy/envoymobile/engine/types/EnvoyErrorCode.java

```diff
@@ -1,17 +0,0 @@
-package io.envoyproxy.envoymobile.engine.types;
-
-public enum EnvoyErrorCode {
```

Comment:
What if we keep this enum, but just move the `EnvoyError` to Kotlin?

That way the code is still enumerated, but the error type that people interact with is still Kotlin.

Reply:
We currently don't have a concept of an `EnvoyErrorCode` on iOS (simply an integer is exposed), so was going for parity here.

I'm totally open to introducing an error code type that we expose on both platforms in Swift/Kotlin if you think that'd be valuable (can either be this PR or a different follow-up).